src/Common/Commands.Common.Test/Commands.Common.Test.csproj

```diff
@@ -314,4 +311,4 @@
     </PropertyGroup>
     <Error Condition="!Exists('..\..\packages\xunit.runner.visualstudio.2.0.0\build\net20\xunit.runner.visualstudio.props')" Text="$([System.String]::Format('$(ErrorText)', '..\..\packages\xunit.runner.visualstudio.2.0.0\build\net20\xunit.runner.visualstudio.props'))" />
```

Comment:
There are some resource files - under Commands.Common.Test\Resources\DSC - that needs to be moved to ServiceManagement.Test project.